lang/test/org/partiql/lang/eval/BindingsTest.kt

```diff
@@ -14,29 +14,29 @@
 
 package org.partiql.lang.eval
 
-import org.junit.Test
-import org.partiql.lang.TestBase
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertNull
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.api.fail
+import org.partiql.lang.ION
+import org.partiql.lang.SqlException
 import org.partiql.lang.errors.ErrorCode
 import org.partiql.lang.util.newFromIonText
 
-class BindingsTest : TestBase() {
+class BindingsTest {
+    private val valueFactory = ExprValueFactory.standard(ION)
 
-    fun bind(text: String): Bindings<ExprValue> = valueFactory.newFromIonText(text).bindings
+    @Test
+    fun delegate() {
+        val innerBindings = valueFactory.newFromIonText("{a:1, b:2}").bindings
+        val outerBindings = valueFactory.newFromIonText("{b:3, c:4}").bindings
 
-    fun over(
-        text: String,
-        bindingsTransform: Bindings<ExprValue>.() -> Bindings<ExprValue>,
-        block: AssertExprValue.() -> Unit
-    ) =
-        AssertExprValue(
-            valueFactory.newFromIonText(text),
-            bindingsTransform
-        ).run(block)
+        val delegatedBindings = innerBindings.delegate(outerBindings)
 
-    @Test
-    fun delegate() = over("{a:1, b:2}", { this.delegate(bind("{b:3, c:4}")) }) {
-        assertBinding("b") { ion.newInt(2) == ionValue }
-        assertBinding("c") { ion.newInt(4) == ionValue }
+        // note that "b" in the inner bindings is shadowed by the "b" in the outer bindings.
+        assertEquals(2, delegatedBindings[BindingName("b", BindingCase.INSENSITIVE)]!!.intValue())
+        assertEquals(4, delegatedBindings[BindingName("c", BindingCase.INSENSITIVE)]!!.intValue())
```

Comment:
This was used old `TestBase` stuff--the refactor tests the same thing but without the dependency on test base.